Assets/Tests_EditMode/EditModeTestScript.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using NUnit.Framework;
+using UnityEditor;
+using UnityEngine;
+using UnityEngine.TestTools;
+using System;
+
+namespace Tests
+{
+    public class EditModeTestScript
+    {
+        // A Test behaves as an ordinary method
+        [Test]
+        public void EditModeTestScriptSimplePasses()
+        {
+            // Use the Assert class to test conditions

```

Comment:
This seems aspirational :)

Reply:
Super-aspirational! I need to add more tests, not sure right now if we need editor tests, since we don't use any custom editor scripts currently, but since they were added automagically I felt that leaving placeholders would be a good hint as to where they need to go in future!